### PR TITLE
tickets/DM-28705

### PR DESCRIPTION
**Pull request by mareuter**
These are some notebooks I added to my area as examples for CSC testing for a training class. No mainline notebooks were touched.

### CODE REVIEW

File: mreuter/CSC_Testing/mt_utils.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
+from astropy.coordinates import AltAz, ICRS, EarthLocation, Angle
+from astropy.time import Time
+import astropy.units as u
+
+from lsst.ts import salobj
+
+def altaz_to_radec(alt, az):
+    location = EarthLocation.from_geodetic(
+        lon=-70.747698 * u.deg, lat=-30.244728 * u.deg, height=2663.0 * u.m
```
Comment (patrickingraham) [inline mreuter/CSC_Testing/mt_utils.py:9]: These should really be in a constants.py file (like we've done for the AuxTel)...

File: mreuter/CSC_Testing/mt_utils.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        az=Angle(az, unit=u.deg),
+        alt=Angle(alt, unit=u.deg),
+        location=location,
+        obstime=current_time
+    )
+
+    return alt_az.transform_to(coord_frame_radec)
+
+async def slew_to_park(mtcs, rotpa=0.0):
+    ra_dec = altaz_to_radec(80.0, 0.0)
```
Comment (patrickingraham) [inline mreuter/CSC_Testing/mt_utils.py:27]: I think this coordinate should either be in a config file, or maybe defined at the top and not buried in the code?

File: mreuter/CSC_Testing/mt_utils.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        dRA=0,
+        dDec=0,
+        rotFrame=1,
+        rotMode=1,
+    )
+
+    await slew(mtcs, True)
+
+async def slew_to_flatfield(mtcs, rotpa=-180.0):
+    ra_dec = altaz_to_radec(39.0, 205.7)
```
Comment (patrickingraham) [inline mreuter/CSC_Testing/mt_utils.py:50]: I think this coordinate should either be in a config file, or maybe defined at the top and not buried in the code?

File: mreuter/CSC_Testing/mt_utils.py [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+    )
+
+    await slew2(mtcs)
+
+async def stop_mt_tracking(mtcs):
+    ack = await mtcs.rem.mtptg.cmd_stopTracking.start(timeout=30.)
+    print("Stop Tracking")
+    print(f"ack={ack.ack} ack.error={ack.error}, ackcmd.result={ack.result}")
+
+async def slew2(mtcs, stop_tracking=False):
```
Comment (patrickingraham) [inline mreuter/CSC_Testing/mt_utils.py:100]: this needs definition such that we know the difference between slew and slew2

**Review by patrickingraham — APPROVED** [review]
Notebooks and utils are all in the personal space. Notebooks appear empty (but please clear them if not).
I started commenting on the utils before realizing they're in a personal space. Sorry. But comments may be useful anyways!

**Comment by mareuter** [comment]
I'm hoping at some point to get rid of the utility functions once the MTCS class has the ones I need.

**Pull request merged**